tests/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
```

Comment:
Curious how the test containers will integrate with GH Actions and it's matrix support